### PR TITLE
Create codecov.yml

### PR DESCRIPTION
**Pull request by tgymnich**
Ignore tests in code coverage report.

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+ignore:
+- "Tests"
+
+codecov:
+  require_ci_to_pass: yes
+
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
```
Comment (bauer-andreas) [inline codecov.yml:11]: I assume this force the code coverage to be between 70-100? Otherwise the PR won't be mergeable or what is the intention?

**Review by bauer-andreas — COMMENTED** [review]
So this is a followup to #49  right?
It basically adds the configuration file for codecov?

File: codecov.yml (hunk 1 above)
Comment (tgymnich) [reply codecov.yml:11]: correct. its just the default value.

Comment (tgymnich) [reply codecov.yml:11]: > So this is a followup to #49 right?
> It basically adds the configuration file for codecov?

yes. Codecov didn't ignore the tests by itself, so I had to add this file

Comment (bauer-andreas) [reply codecov.yml:11]: Does the CI run enforce a certain code coverage? Or does the CI job just fail when the code coverage decreases like in this case?

Comment (tgymnich) [reply codecov.yml:11]: it just fails. Codecov checks are not "required" Should we set the range to 51% (current coverage) or leave it a 70 to motivate us to write tests?

**Review by bauer-andreas — DISMISSED** [review]

**Review by bauer-andreas — APPROVED** [review]

**Pull request merged**